ddr/tools/getmacros

```diff
@@ -200,6 +202,9 @@ parse_namespace_policy_files() {
 	local filenum=0
 
 	for f in "${annotated_files[@]}"; do
+		#remove any double // which comments out the rest of the line in C/C++
+		f=${f//\/\//\/}
```

Comment:
This doesn't look right. I think it should be `${f/\/\//\/}`, but that would still only do one substitution.
These are pathnames in the output of `grep -l`; perhaps a better solution would be to add
`| sed -e 's|//+|/|g'` to the end of the pipeline that produces `annotated_files`.

Reply:
The two slashes at the start matches all instances.

From https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion :
```
${parameter/pattern/string}
The pattern is expanded to produce a pattern just as in filename expansion. Parameter is expanded and the longest match of pattern against its value is replaced with string. If pattern begins with ‘/’, all matches of pattern are replaced with string. Normally only the first match is replaced. 
```

Reply:
Thanks, I missed that subtlety as I read the bash manual.
I'm still unclear how we wind up with paths containing `//`.

Reply:
We pass in `../` as the parameter for the top level search directory. In the `get_all_annotated_files` function, it looks like OSX doesn't "fix" the path and just leaves the extra slash in.

Reply:
Perhaps it would be better to remove any trailing slash from that directory rather than apply a fix to every file name.